### ISSUE TITLE
Granade storage

### ISSUE THREAD

**Issue by jakub913** [splewis/csgo-practice-mode#182]
Hello, I am facing a problem with practise mode, everything is working great except saving granades. I have a latest version and everything set up. When I save a smoke, nothing in my data storage happens. i have here just backups and .gitignore file, everything empty. This happens when I close the server, until then everything is saved, but still it is not in my database, so I cant save it manually. So when I close the server I lose everything. Please help me find a solution, I really need help. Thanks

**Comment by jakub913** [comment]
Turned out that I have to succesfully disconnect from the server, because granades are saving after you disconnect, when you write .save xxx and not disconnect normally, your granades are not gonna be saved.

**Comment by splewis** [comment]
Yes, grenades are only flushed back to files when a client disconnects, the map changes, or the plugin is unloaded, by design.

How problematic is that for you? Technically it could flush after every edit - or at least every minute or so. PRs welcome on those.

**Comment by chriskhosh** [comment]
Can you add a way to save (flush) all nades by admin command, such as .flush or something similar?
